Tests/Twig/Extension/PageExtensionTest.php

```diff
@@ -66,12 +66,14 @@ public function testController()
         ;
         $extension = new PageExtension($cmsManager, $siteSelector, $router, $blockHelper, $HttpKernelExtension);
         $extension->initRuntime($env);
-        $HttpKernelExtension->expects($this->once())->method('controller')->with(
-            'foo',
-            array('pathInfo' => '/foo/bar/'),
-            array()
-        )
-        ;
+        if (!method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
+            $HttpKernelExtension->expects($this->once())->method('controller')->with(
```

Comment:
@greg0ire can you rename the variable to `$httpKernelExtension` (starting lowercase instead of uppercase) ?